openapi/wordpress/types.bal

```diff
@@ -0,0 +1,811 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+public type PostArr Post[];
+
+public type RevisionArr Revision[];
+
+public type PageArr Page[];
+
+public type MediaArr Media[];
+
+public type TypeArr Type[];
+
+public type StatusArr Status[];
+
+public type CommentArr Comment[];
+
+public type TaxonomyArr Taxonomy[];
+
+public type CategoryArr Category[];
+
+public type TagArr Tag[];
+
+public type UserArr User[];
+
+# Type of resource.
+public type FormdataMediaType string;
+
+# Maximum number of items to be returned in result set. Default 10
+public type FormdataPerPage int;
+
+# Current page of the collection. Default 1
+public type FormdataPage int;
+
+public type Body12 record {
```

Comment:
This connector might be generated using an older version of Ballerina OpenAPI tool. Please try Ballerina Beta3 RC 4 or RC 5 pack.

Reply:
Updated as suggested. Please check @aneeshafedo